prisma/seed.ts

```diff
@@ -300,6 +300,24 @@ seed('Default projects', async () => {
                     },
                 });
             }
+
+            const [goal] = allGoals;
+
+            // eslint-disable-next-line no-await-in-loop
+            await prisma.goalHistory.create({
+                data: {
+                    goalId: goal.id,
+                    subject: 'state',
+                    action: 'change',
+                    previousValue: 'InProgress',
+                    nextValue: 'Critical',
+                    activityId: sample(allUsers).activityId,
+                },
+                include: {
```

Comment:
Unused values. `include` makes sense if you are using result of req, but here you don't.